rustls-symcrypt/Cargo.toml

```diff
@@ -15,7 +15,12 @@ readme = "README.md"
 # Disable aws_ls_rc since it is not needed and increases build time.
 rustls = { version = "0.23.0", features = ["ring", "tls12", "std"], default-features = false }
```

Comment:
The ring feature should be removed, as this PR removes the last usage of ring.

Reply:
removed. Thanks for finding it!